README.md

```diff
@@ -1,6 +1,25 @@
 # superset-k8s-charm
 Superset is fast, lightweight, intuitive, and loaded with options that make it easy for users of all skill sets to explore and visualize their data.
 
+## Superset UI, worker and beat deployment
+The same Superset charm can act as either a web server, worker or beat-scheduler. This is determined by the configuration parameter `charm-function`, which can be any of `app-gunicorn` (web), `app` (web development), `worker` or `beat`. Each Superset application must be related to the same PostgreSQL and Redis clusters for communication. The default value of `charm-function` is `app-gunicorn`.
+
+```
+# pack the charm
+charmcraft pack
+
+# deploy the web server
+juju deploy ./superset-k8s_ubuntu-22.04-amd64.charm --resource superset-image=apache/superset:2.1.0 superset-k8s-ui
+
+# deploy a worker
+juju deploy ./superset-k8s_ubuntu-22.04-amd64.charm --resource superset-image=apache/superset:2.1.0 --config charm-function=worker superset-k8s-worker
```

Comment:
The ```README``` should likely reflect deployment instructions which will be available on the Charmhub homepage. So I would change this to:
```juju deploy superset-k8s --config charm-function=worker superset-k8s-worker```

Reply:
I was planning on updating this when the charm was ready to be published to charmhub. At the moment I feel this would be misleading as someone who comes across this repository could not run this command to deploy superset at this time.